src/Helpers/Hooks.js

```diff
@@ -312,3 +313,51 @@ export const useAccountStats = (setLoading, addNotification) => {
 
     return { hasEdgeDevices, hasConventionalSystems, includesCvesWithoutErrata, isAdvisoryAvailable, hasAccess };
 };
+
+export const useHccEnvironmentContext = () => {
+    const chrome = useChrome();
+
+    const [[
+        canAccessApplication,
+        canExport,
+        canEditPairStatus,
+        canReadExcludedSystem,
+        canEditSystemExclusion,
+        canEditStatusAndBusinessRisk,
+        canDoAdvancedReporting,
+        canToggleCvesWithoutErrata,
+        canReadInventory
+    ], isRbacLoading] = useRbac([
+        PERMISSIONS.accessApplication,
+        PERMISSIONS.basicReporting,
+        PERMISSIONS.setPairStatus,
+        PERMISSIONS.readExcluded,
+        PERMISSIONS.setExcludedIncluded,
+        PERMISSIONS.setCveStatusAndBusinessRisk,
+        PERMISSIONS.advancedReporting,
+        PERMISSIONS.toggleCvesWithoutErrata,
+        PERMISSIONS.readHosts
+    ]);
+
+    return {
+        isLoading: isRbacLoading,
+        isApplicationAccessible: canAccessApplication,
+        isStatusDisplayed: true,
+        isBusinessRiskDisplayed: true,
+        isRemediationEnabled: true,
+        areExcludedSystemsDisplayed: canReadExcludedSystem,
+        areSystemsAccessible: canReadInventory,
+        isAdvancedReportingAccessible: canDoAdvancedReporting,
+        isExportEnabled: canExport,
+        isSystemExclusionEditable: canEditSystemExclusion,
+        isPairStatusEditable: canEditPairStatus,
+        isStatusEditable: canEditStatusAndBusinessRisk,
+        isBusinessRiskEditable: canEditStatusAndBusinessRisk,
+        areCvesWithoutErrataTogglable: canToggleCvesWithoutErrata,
+        updateDocumentTitle: chrome.updateDocumentTitle,
+        getUser: chrome.auth.getUser,
+        on: chrome.on,
+        hideGlobalFilter: chrome.hideGlobalFilter,
+        mapGlobalFilter: chrome.mapGlobalFilter
```

Comment:
What about just
```suggestion
        ...chrome
```
?

Reply:
I'm worried that with this approach we might use some Chrome function in HCC context and forget to implement/mock it in IoP and it will lead to errors.

But I will consider it in the future.